### PR TITLE
Added more algorithms to STINGER

### PR DESCRIPTION
**Pull request by argonjules**
Temporary placeholder.  Still working on this.  Using PR to check TravisCI build.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -143,6 +143,10 @@ add_test(StingerTraversalTest ${CMAKE_BINARY_DIR}/bin/stinger_traversal_test)
 add_test(StingerPagerankTest ${CMAKE_BINARY_DIR}/bin/stinger_pagerank_test)
 add_test(StingerAdamicAdarTest ${CMAKE_BINARY_DIR}/bin/stinger_adamic_adar_test)
 add_test(StingerBetweennessTest ${CMAKE_BINARY_DIR}/bin/stinger_betweenness_test)
+add_test(StingerHitsTest ${CMAKE_BINARY_DIR}/bin/stinger_hits_test)
+add_test(StingerHitsTest ${CMAKE_BINARY_DIR}/bin/stinger_diameter_test)
```
Comment (davidediger) [inline CMakeLists.txt]: I think you want these to have unique names like "StingerDiameterTest", etc.

Comment (argonjules) [inline CMakeLists.txt]: oops that's a typo - I'll fix that now

**Comment by tgoodyear** [comment]
Can these be squashed down to a single commit whose commit message indicates the names of algorithms that have been added?

**Comment by sirpoovey** [comment]
Let's hold off on the rebase until the work is complete

File: src/clients/algorithms/community_detection/src/community_detection.c [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+
+    /* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ *
+    * Setup and register algorithm with the server
+    * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ */
+    stinger_registered_alg * alg =
+    stinger_register_alg(
+            .name=alg_name,
+    .data_per_vertex=sizeof(int64_t),
+    .data_description="d partitions",
```
Comment (rmfoard) [inline src/clients/algorithms/community_detection/src/community_detection.c]: data_description should be set to "l partitions" -- the values are integral.

**Comment by davidediger** [comment]
Okay @argonjules it's time to rebase and squash down to a single commit.  Please include the names of the new algorithms in the commit message.

**Comment by davidediger** [comment]
Merged in #206

**Pull request closed without merging**